osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -101,21 +101,6 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
 
 
 - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication *)sender {
```

Comment:
I think we can remove this handler now?  It doesn't seem to do anything anymore.